src/main/java/net/projectzombie/region_rotation/commands/RRText.java

```diff
@@ -108,22 +109,24 @@ static public String formatPermission(final StateController controller, final St
     ROTATE_BASESTATE_PERM       = "rotate",
     INFO_BASESTATE_PERM         = "info",
     LIST_BASESTATE_PERM         = "list",
+    CHANGE_BROADCAST_PERM       = "cbroadcast",
 
     CREATE_CONTROLLER_PERM      = "controller.create",
     DESTROY_CONTROLLER_PERM     = "controller.destroy",
     LIST_CONTROLLER_PERM        = "controller.list",
 
-    STATE_COMMAND_LIST          = "<create:addalt:destroy:removealt:reset:rotate:info:list>",
+    STATE_COMMAND_LIST          = "<create:addalt:destroy:removealt:reset:rotate:info:list:cbroadcast>",
     CONTROLLER_COMMAND_LIST     = "<" + formatControllerName("cName") + ":create:destroy:list>",
 
-    ADD_BASESTATE_HELP          = "create <rName> <world> <backup rName> <backup world>",
-    ADD_ALT_BASESTATE_HELP      = "addalt <rName> <alt rName> <alt world>",
-    REMOVE_BASESTATE_HELP       = "destroy <rName>",
-    REMOVE_ALTSTATE_HELP        = "removealt <rName> <alt rName>>",
-    RESET_BASESTATE_HELP        = "reset <rName> <broadcast = F>",
-    ROTATE_BASESTATE_HELP       = "rotate <rName> <alt rName> <rotate air = T> <broadcast = F>",
-    INFO_BASESTATE_HELP         = "info <rName>",
-    LIST_BASESTATE_HELP         = "list",
+    ADD_BASESTATE_HELP          = "<cName> create <rName> <world> <backup rName> <backup world>",
```

Comment:
Do not include <cName>, it will display automatically using helper functions defined above.